project.toml

```diff
@@ -0,0 +1,2 @@
+[build-system]
```

Comment:
Same comments as above. Though if it is possible to get rid of the duplication of toml files in this PR that's an even better win.

Reply:
Yes, I thought the same, even using a `setup.cfg` for metadata deduplication as well. I will probably use a symlink from bindings/python and keep just the top one as a regular file. For setup.py that is not possible without some adding some more hacks, so that I'd like to keep separate because the setup of RPATH and CMake flags propagation is different.